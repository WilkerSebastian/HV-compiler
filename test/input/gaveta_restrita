0-100 100 000