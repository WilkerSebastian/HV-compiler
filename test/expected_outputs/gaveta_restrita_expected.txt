Erro tentativa de sobrescrita de gaveta que armazena código fonte conteúdo da gaveta(0): 0-100
